src/silx/app/compare/main.py

```diff
@@ -70,9 +70,9 @@ def mainQt(options):
         logging.root.setLevel(logging.DEBUG)
 
     if options.use_opengl_plot:
-        backend = "gl"
+        backend = "opengl", "matplotlib"
```

Comment:
You can't specify 2 backends here right?

Reply:
Yes you can!

But, I'll update it to fore OpenGL when requested from command line.